ktor-http/common/src/io/ktor/http/URLBuilder.kt

```diff
@@ -169,6 +169,11 @@ private fun <A : Appendable> URLBuilder.appendTo(out: A): A {
             out.appendTel(host)
             return out
         }
+
+        "data" -> {
+            out.appendAbout(host) // We use the same append format as about URLs
```

Comment:
I think we could rename `appendAbout` into something more general and remove `appendTel`

Reply:
This part is called payload in the spec, so I will rename it to appendPayload